endpoints/api/test/test_capabilities.py

```
@@ -0,0 +1,94 @@
+from endpoints.test.shared import client_with_identity, toggle_feature
+from test.fixtures import *
+
+
+class TestRegistryCapabilities:
+    def test_capabilities_no_auth_required(self, app):
+        """Test capabilities endpoint accessible without authentication."""
+        from endpoints.api.capabilities import RegistryCapabilities
+        from endpoints.test.shared import add_csrf_param
+
+        with app.test_client() as cl:
+            params = add_csrf_param(cl, {})
```

Ruff (0.14.11)

Local variable `params` is assigned to but never used

Remove assignment to unused variable `params`

(F841)